docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Win32 App Isolation Introduction
+
+### Target Applications
+
+* Win32
+* Centennial
+
+### Minimum requirements:
+
+* Windows insider version >= 25357
+* [Customized MSIX Packaging Tool](https://github.com/microsoft/win32-app-isolation/releases/tag/v0.1.1) This version is required to build a Win32 App Isolation application
+* If is needed to know which capabilities are required, one can use [ACP](https://github.com/microsoft/win32-app-isolation/releases/tag/v0.1.1) and [WPR](https://learn.microsoft.com/en-us/windows-hardware/test/wpt/windows-performance-recorder)
+
+### Creating a Silo App:
+
+1. Create an MSIX package from Win32 installer *if the app is not already MSIX* ([step 1](docs/packaging/msix-packaging-tool.md#win32---msix))
```

Comment:
The link should be `[step 1](packaging/msix-packaging-tool.md#win32---msix)`, my bad. (or `[step 1](/docs/packaging/msix-packaging-tool.md#win32---msix)`). Same to the links below. +1 for the link checker action.